pkg/policy/opa/engine_test.go

```diff
@@ -165,13 +166,25 @@ func TestFilterRules(t *testing.T) {
 			assert:      true,
 			regoMapSize: 7,
 		},
```

Comment:
@gaurav-gogia regarding the test coverage, you need to cover all the failure test cases ( such as invalid category values, etc.. ) , you need to cover the test cases for the precedence of the flag values over the values passing in config.toml files.. , and you need to cover all the edge cases you can think of.

Reply:
will look into it